src/main/resources/co/phoenixlab/discord/resources/locale.properties

```diff
@@ -323,6 +323,7 @@ commands.dn.crit.response.format.required           = **Level {0|%d} Critical Ch
 commands.dn.crit.response.invalid                   = Invalid format. Use `{0|%s}[commands.general.dn.command] \
   [commands.help.command] [commands.dn.crit.command]` for more information
 commands.dn.crit.response.level_out_of_range        = Level must be between {0|%d} and {1|%d} inclusive
+commands.dn.critdmg.response.low_percent 			= Critical chance percentage specified must be greater than 0%
```

Comment:
This is the same key as the one below, should be `commands.dn.crit.response.low_percent`